### DISCUSSION TITLE
README.MD is less easy to understand

### DISCUSSION THREAD

**Discussion by aryapramudika** [ngadmini/Grabbing-Blacklist-for-Bind9-RPZ#135] [Q&A] [answered]
After running grab_http, what to do after that? the README say then "follow the next step". What is next step?

How to running this script direct on server?

**Comment by ngadmini** [comment, marked as answer]
Halo Bro @aryapramudika 

> After running grab_http, what to do after that? the README say then "follow the next step". What is next step?

**Next step** muncul setelah [grab_http.sh](https://github.com/ngadmini/Grabbing-Blacklist-for-Bind9-RPZ/blob/master/libs/grab_http.sh) selesai. seperti di [wiki](https://github.com/ngadmini/Grabbing-Blacklist-for-Bind9-RPZ/wiki#the-output-display)
```
[DONE] completed IN 1m:30s:836ms
[WARN] there are still remaining :
   - duplicate entries across CATEGORIES
   - sub-domains and it's parent-domain across CATEGORIES
   - ipv4 that can be reduced to a CIDR-block
[HINT] consider continue to next TASKs
you have 4 options to proceed to the next task:
   1) pruning ipv4, duplicate & sub-domain entries across CATEGORIES
   2) option [1] and rewriting ipv4 & domain entries to RPZ format-entry [db.* files]
   3) option [2] and incrementing serial zones [rpz.* files]
   4) option [3] and syncronizing latest [rpz.* and db.*] files to rpz.warnet-ersa.net
      WARNING: remote-host rpz.warnet-ersa.net will REBOOT due to low memory
               use 'shutdown -c' at remote-host: rpz.warnet-ersa.net to abort
  ^C) use (Ctrl+C) to quit
please hit: (1|2|3|4) to continue OR (Ctrl+C) to quit
```



> How to running this script direct on server?

Tidak bisa bro, mesti dilakukan secara remote dari desktop Linux. Kalau pun bisa, harus dilakukan banyak perubahan. 

Terima kasih
